src/filters/simple-filter.html

```diff
@@ -2,8 +2,8 @@
   <form>
     <div class="form-group">
       <div class="input-group">
-        <div class="input-group-btn">
-          <button type="button" class="btn btn-default dropdown-toggle filter-fields" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        <div dropdown class="input-group-btn">
+          <button dropdown-toggle type="button" class="btn btn-default dropdown-toggle filter-fields" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
I agree with this change but we need to add angular-bootstrap as a devDependency I believe.